decidim-admin/app/controllers/decidim/admin/newsletters_controller.rb

```diff
@@ -11,6 +11,7 @@ class NewslettersController < Decidim::Admin::ApplicationController
 
       def index
         enforce_permission_to :index, :newsletter
+        @recipients_count = recipients_count_query(form(SelectiveNewsletterForm).instance)
```

Comment:
`recipients_count` can be a helper_method used in the controller and views, and then theres no need tho declare a global variable here and in other actions of this controller and so DRY up the code a little. :smile:

Reply:
:arrow_up:  don't you think so?